game/src/Game/GameSession/BillboardManager.gd

```diff
@@ -1,23 +1,31 @@
 extends MultiMeshInstance3D
 
-#given a name: get the index for the texture in the shader
-# this is to reduce the number of magic indeces in the code
-# to get the proper billboard image
-var billboard_names : Dictionary = {}
-
 @export var _map_view : MapView
 
-const SCALE_FACTOR : float = 1.0/96.0
-
-enum ProvinceBillboards { NONE, INVISIBLE, RGO, CRIME, NATIONAL_FOCUS }
+const SCALE_FACTOR : float = 1.0 / 96.0
+const ICON_DIMS : Vector2 = Vector2(1.0, 0.75)
+const CAPITAL_DIMS : Vector2 = Vector2(1.0, 1.0)
+
+enum ProvinceBillboards { NONE, INVISIBLE, RGO, CRIME, NATIONAL_FOCUS, CAPITAL }
```

Comment:
The original intent was to represent the current state of the province billboards. Now that it also includes CAPITAL, it should be renamed something more generic like `BillboardState`. It doesn't have to be that name, just something better descriptive of what the enum is for.